server_utils/origin.go

```diff
@@ -758,11 +784,29 @@ func CheckOriginSentinelLocations(exports []OriginExport) (ok bool, err error) {
 			if path.Base(sentinelPath) != sentinelPath {
 				return false, errors.Errorf("invalid SentinelLocation path for StoragePrefix %s, file must not contain a directory. Got %s", export.StoragePrefix, export.SentinelLocation)
 			}
-			fullPath := filepath.Join(export.StoragePrefix, sentinelPath)
-			_, err := os.Stat(fullPath)
+
```

Comment:
After testing some of this by hand, I'm realizing we don't have a single log statement saying that this check is being performed. Can you add something along the lines of `log.Infof("Checking that sentinel object %s is accessible for federation prefix %s", sentinelPath, federationPrefix)` to this loop so the user can see that each object is being queried?